Tutorials/Editors-Manual/Getting-Started-With-Umbraco/Editing-Existing-Content/index.md

```diff
@@ -1,34 +1,65 @@
 ---
-versionFrom: 7.0.0
-needsV8Update: "true"
+versionFrom: 8.0.0
 ---
 
 # Editing Existing Content
 
 ## Content Within the Tree View
 
-When you want to edit content on an existing page you can follow this process:
+This documentation is intended as a guide that will be helping you gain an overview of how you edit in excisting content.
 
-1. Find and select the page you want to edit.
-2. Edit the pages content within the content area. Each different type of page will have different content fields for you to populate such as text boxes, content and date pickers and rich text editors. See section 3 for more information on working with content.
-3. Click the arrow on the ***Save and publish*** button to reveal more options. Now you can click ***Save*** in the popup menu.
-4. Click the ***Preview*** button to preview how the page will look once published to the live website.
-5. If you are happy with the changes you can now click ***Save and publish*** which will make the changes live.
+1.	When you are looking to edit content, the first thing you should do is locate the ***page*** you want to edit content on. This can be done in the section tree on the left side of the screen.
 
-![editPage.jpg](images/editPage.jpg)
+![Pick the page you want to edit in the section tree.](images/Pick-page.png)
+
+2.	When you have located the page, you want to ***edit*** and clicked onto it.
+You will be presented with the content of the page (if there is any). 
+This content will be in the middle of the page (see image) the content is everything inside the red box, in this case there is three kinds of content.
+- A text box for the title.
+- A media picker for the image.
+- A rich text editor.
+
+These are just three of many kinds of content types.
+
+![This shows the content on the middle of the page.](images/Edit-content.png)
+
+3.	After you have edited the content, you can click the save button to ***save*** the edits you did without publishing it.
+
+![This shows the save button in the buttom left of the screen.](images/Save.png)
+
+4.	At this point it is recommended that you ***preview*** the changes that you did, which you can do by pressing the preview button located to the left of the save button.
+
+![This shows the preview button next to the save button.](images/Preview.png)
+
+5.	Finally, after you have done the preview and confirmed that the changes are how you like it.
```

Comment:
This sentence is a bit wierd. Wouldn't it make more sense, if this one if merged with the one on the next line? 🤔

```suggestion
5.	Finally, after you have done the preview and confirmed that the changes are how you like it.
```